web_src/js/features/common-global.js

```diff
@@ -142,16 +142,12 @@ export function initGlobalCommon() {
     }
   });
 
-  // loading-button this logic used to prevent push one form more than one time
-  $(document).on('click', '.button.loading-button', function (e) {
-    const $btn = $(this);
-
-    if ($btn.hasClass('loading')) {
-      e.preventDefault();
-      return false;
-    }
-
-    $btn.addClass('loading disabled');
+  // prevent multiple form submissions on forms containing .loading-button
+  document.addEventListener('submit', (e) => {
+    const btn = e.target.querySelector('.loading-button');
```

Comment:
Sorry, but I'm not going to write jQuery just because it "fits" better.

Reply:
I am fine with it since it's a full function without other dependency.

I just have objection to something like  `el.setAttribute(...); $(el).attr(....)`.